.github/workflows/lint-project.yaml

```diff
@@ -0,0 +1,32 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
```

Comment:
This file is autogenerated by using the tool  `usethis::use_github_action("lint-project")`